extension/chrome/content/download.js

```diff
@@ -36,7 +36,9 @@
 
 var Cc = Components.classes;
 var Ci = Components.interfaces;
+var Cu = Components.utils;
 
+Cu.import("resource://gre/modules/Downloads.jsm");
```

Comment:
Sure but that can be done in the release issue. The bump to v26 has to happen with this PR.